### ISSUE TITLE
FTU Explorer future feature request

### ISSUE THREAD

**Issue by supriyabidanta** [hubmapconsortium/hra-ui#348]
@LibbyUX  @bherr2 
Changes that could make the FTU explorer look better:

1. Can we make the FTU library shrinkable?
2.  same with the biomarker section
3. A clickable icon to zoom in or zoom out from the FTU illustration.

Attached is an image for your reference:

![Screenshot 2024-04-15 at 1 44 14 PM](https://github.com/hubmapconsortium/hra-ui/assets/117299113/bbd84784-ea6e-4e2a-aa68-f8b94144e340)

**Comment by LibbyUX** [comment]
Thank you @supriyabidanta for stopping by my desk and for submitting these details! 

Bruce and I will discuss what we can do in 2024. We will keep your feature requests documented here for safekeeping, for now! Thanks for reaching out and getting this conversation started!

**Comment by LibbyUX** [comment]
# Supriya & Libby meeting notes!
Thanks for the great lunch and meeting today, @supriyabidanta! 

## Zoom feature notes
Check feasibility with Bruce for potential options for zooming into the illustration:
- Drag to select zoom area
- Magnify button with a plus and minus
- Pan on drag on zoomed in image

## Other features covered:
- Collapse FTU listing to give more space for illustration
- Fixed width for right side matrix & source data
  - Source data collapse or default small logic for small amounts of source data
- Potential menu consolidation for embedding illustration, downloading, DOI metadata, and downloading data
  - Note that download data button should not show if we don't have data for the illustration

**Comment by bherr2** [comment]
@LibbyUX Katy is requesting to add dataset metadata including volume, sex, etc. She is thinking we may need to add more metadata to the bottom right table.

**Comment by LibbyUX** [comment]
@bherr2 yes, I think we're going to need a filter for the source data table eventually. Below is an early iteration of the entry point. I see this being a modal overlay with filter options for the table. Let me know if that's feasible!

<img width="417" alt="Screenshot 2024-08-08 at 12 42 14 PM" src="https://github.com/user-attachments/assets/0535e7a6-9cc0-4578-8eb0-8766b142c367">

**Comment by LibbyUX** [comment]
# FTUs with FIBI images: 9th release feature request

@supriyabidanta and I discuss adding FIBI images to the FTU Explorer for 9th release:
- About 10 FTUs could link to relevant FIBI images (GIF) for analysis
- Currently, the FIBI GIFs are hosted elsewhere (see resources below).
  - Supriya and I discussed routing users to those URLs, but we may want to design/build UI for it. More information is needed.

In the app, this feature request could look like:
- An icon button next to the FTU name in the FTU Library list
- A picture in picture + button/menu that routes out in the illustration panel

Before more work begins: More information is needed on the UI vision from @katyb. Infrastructure expectations from @bherr2 also must be better understood.

### Resources
- FIBI image from Supriya: https://ucdavis.app.box.com/s/uud09avu64ebv3uoly8l8jzehzgwdxx9/file/1662710560707

### Low fidelity mockups
![image](https://github.com/user-attachments/assets/b1b4be63-fbfa-48b9-8a66-632f56c810d6)

**Comment by LibbyUX** [comment]
# FIBI feature resources
@supriyabidanta, @danqaurooni, Nate Anderson (UC Davis Pathology), Rishi and I meet yesterday at 3:00pm:
- [Meeting notes](https://docs.google.com/document/d/1EUjzAQnl91eNRfkeSg8MI93IOoY9dKvxPJajphdIXc4/edit?tab=t.0#heading=h.v28vmtxmr738) FIBI image meeting
- [Figma design file iterations](https://www.figma.com/design/VlNjSrctLEc0Wyu32FmnZq/Functional-Tissue-Unit-(FTU)-Explorer?node-id=6578-37540&t=Hxw3iIemm7lHZLI1-4)
- [Path Presenter example](https://pathpresenter.net/public/display?token=35aae6be) featuring similar functionality as desired

**Comment by LibbyUX** [comment]
# Feature request from Katy 1.28.2025 via email:
![Image](https://github.com/user-attachments/assets/6e4304cb-5b51-4d6c-8394-d5fdd48a09d3)
